src/content/admin/okteto-api.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Okteto API Documentation
+description: Programmatic access to resources managed by Okteto
+sidebar_label: Okteto API
+id: okteto-api
+---
+
+import Image from "@theme/Image";
+
+The Okteto API `BETA` is defined using OpenAPI, providing access to resources managed by Okteto.
+Combined with the Okteto CLI or other tools, it allows you to automate workflows, such as deleting unused development volumes within a Namespace or redeploying applications.
+
+:::note
+🚧 This API is currently in beta
+- Endpoints, request/response formats, and authentication mechanisms are subject to change as we refine the API
+- Backward-incompatible changes may occur before the API reaches general availability (GA)
+
+For feedback or issues, contact us at support@okteto.com.
+:::
+
+## Getting Started
+The Okteto API `BETA` is accessible with an [Admin Access Token](admin/dashboard.mdx#admin-access-tokens)
+
+To access the full Swagger API documentation, follow these steps:
+	1.	Navigate to **Admin → Admin Access Tokens** in the Okteto Dashboard
+	2.	Click on the link to the **Public API Reference**
+
+The Swagger documentation provides details on all available endpoints, request formats, and response structures.
+You can even run live tests to example responses for your query.
+
+## Overview
+
+The Okteto API allows you to programmatically manage resources such as Namespaces, applications, and other platform-level configurations. In conjunction with the CLI, it can be used to automate tasks like:
+- [Querying Namespace and application statuses](https://github.com/okteto-community/namespaces-report-csv)
+- [Redeploying applications to keep them up to date](https://github.com/okteto-community/redeploy-applications)
+- [Sleep Namespaces based on your own custom settings](https://github.com/okteto-community/sleep-namespaces)
+- A full list of our examples can be found on [GitHub here](https://github.com/topics/okteto-api)
+
+### Authentication
+
+All requests to the Okteto API require authentication via Bearer Tokens.
+You must include your Admin Access Token in the Authorization header for each request.
+
+### Available Endpoints
+
+🚀 For a complete list of API endpoints, refer to the Swagger API Documentation.
+
+The Okteto API includes operations for:
+- Namespaces → List and manage Development and Preview Environments
+- Applications → Deploy and track applications within Namespaces
```

Comment:
Reading this I got the impression the API allows me to deploy an application, and it is not the case. Should we just say `Track applications within namespaces`?